Lib9c/Model/WorldInformation.cs

```diff
@@ -397,6 +389,21 @@ public bool TryGetLastClearedStageId(out int stageId)
             _worlds[worldId] = new World(world, clearedAt, stageId);
         }
 
+        public void AddNewWorld(WorldSheet.Row worldRow, long unlockedAt, WorldSheet worldSheet)
+        {
+            var worldId = worldRow.Id;
+            if (IsStageCleared(worldRow.StageBegin - 1))
+            {
+                var world = new World(worldRow);
+                _worlds.Add(worldId, world);
+                UnlockWorld(worldId, unlockedAt, worldSheet);
+            }
```

Comment:
If it was to unlock while adding `world`, it would have been better to have a detailed name like `AddNewWorldAndUnlock()`. However, I can guess because there is `unlockedAt` in the argument.